config/core/test-configs.yaml

```diff
@@ -174,10 +174,21 @@ test_plans:
     params:
       job_timeout: '60'
       docker_image: 'kernelci/chromeos-flash'
-      flashing_kernel: https://storage.chromeos.kernelci.org/images/kernel/v5.10.112-x86-chromebook/kernel/bzImage
-      flashing_modules: https://storage.chromeos.kernelci.org/images/kernel/v5.10.112-x86-chromebook/modules.tar.xz
-      flashing_rootfs: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/full.rootfs.tar.xz
-      flashing_ramdisk: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/initrd.cpio.gz
+      chromeos_image: 'chromiumos_test_image-R94-14150.19.0-octopus.bin'
+      flashing_kernel: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64+x86_chromebook/kernel/bzImage
+      flashing_modules: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64+x86_chromebook/modules.tar.xz
+      flashing_rootfs: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220505.0/bullseye-chromeos-flash/amd64/full.rootfs.tar.xz
+      flashing_ramdisk: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220505.0/bullseye-chromeos-flash/amd64/initrd.cpio.gz
+
+  chromeos-install-modules:
+    pattern: 'chromeos/chromeos-install-modules-template.jinja2'
+    params:
+      job_timeout: '60'
+      modules_url: http://images.collabora.com/lava/chromeos/modules-4.14.243.tar.gz
```

Comment:
The modules should be populated from the build meta-data.  We could do something special for "fixed kernels" like with QEMU but it seems this would be a follow-up. So I think we should drop this for now.